lms/djangoapps/branding/api.py

```diff
@@ -215,6 +215,7 @@ def _footer_navigation_links():
             #("blog", marketing_link("BLOG"), _("Blog")),
             #("news", marketing_link("NEWS"), _("News")),
             #("help-center", settings.SUPPORT_SITE_LINK, _("Help Center")),
+            #("FerieBezNudy", "/FerieBezNudy", _("#FerieBezNudy")),
```

Comment:
wywalić a nie komentować

Reply:
nie wywalamy, bo 18 stycznia trzeba odkomentować, na chwilę obecną poszła prośba o wyłączenie z footera linka

Reply:
tak, potwierdzam, prosiłam o zakomentowanie, w poniedziałek będzie fix który to odkomentuje.